zulip_bots/zulip_bots/bots/file_uploader/file_uploader.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from pathlib import Path
+
+class FileUploaderHanlder(object):
+    def usage(self) -> str:
+        return (
+            'This interactive bot used to upload files, images, etc to Zulip server.\n'
+            'Specify the file path you want to upload and file will be upload to Zulip server'
+        )
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
```

Comment:
The type of `message` can be more specific than `Any`, though I'm aware that the typing of many bots in the repo is quite loose in this way; that said, I'm not convinced the bot will be type-checked automatically at the moment, since you don't have a diff to `tools/run-mypy` to explicitly check it?

Reply:
It seems to be a my mistake. Changed type to more precise `Dict[str, str]` as in another bots.